newrelic_lambda_cli/integrations.py

```diff
@@ -405,7 +405,7 @@ def create_integration_role(input):
         role = _get_role(input.session, role_name)
         success(
             "Created role [%s] with policy [%s] in AWS account."
-            % (role_name, input.role_policy)
+            % (role_name, input.role_name)
```

Comment:
If we're substituting `role_name` and `input.role_name` in this string...should we rewrite the string? It seems to imply that the second term is going to be policy-related, rather than a role-related value. Also...do we expect `role_name` and `input.role_name` to have different or equivalent values?